src/main/resources/application.yaml

```diff
@@ -9,3 +9,11 @@ management:
 spring:
   application:
     name: SscsBulkScan
+
+ccd:
+ case:
+  jurisdictionId: ${CCD_CASE_JURISDICTION_ID:SSCS}
```

Comment:
no need to add those env vars spring will automatically convert to:
`CCD_CASE_JURISDICTIONID` not positive on whether it will insert an _ between jurisdiction and ID
But I would suggest dropping the ID anyway jurisdiction is enough imo

Reply:
sure will drop id.